keyboards/gmmk/gmmk2/p96/src/eeprom_flash.c

```diff
@@ -0,0 +1,692 @@
+/* Copyright (C) 2021 Westberry Technology (ChangZhou) Corp., Ltd
```

Comment:
Please remove this file, it has been made redundant by the wear-leveling system.